modes/longitudinal/src/index.js

```diff
@@ -13,25 +14,31 @@ const tracked = {
   viewport: 'org.ohif.measurement-tracking.viewportModule.cornerstone-tracked',
 };
 
-const dicomsr = {
-  sopClassHandler: 'org.ohif.dicom-sr.sopClassHandlerModule.dicom-sr',
-  viewport: 'org.ohif.dicom-sr.viewportModule.dicom-sr',
-};
 
-const dicomvideo = {
-  sopClassHandler: 'org.ohif.dicom-video.sopClassHandlerModule.dicom-video',
-  viewport: 'org.ohif.dicom-video.viewportModule.dicom-video',
-}
+const { longitudinalMode } = ConfigPoint.register({
```

Comment:
@swederik and @sedghi  - note how the mode itself is configured with config point, which was necessary in order to allow injecting viewports and sop class handlers.  This means that the mode is no longer a global object, and can be added/created dynamically by choosing what code to import.    This is the other piece that I wanted Erik to see.
Other than these two items, I think the rest can be reviewed by other people.